clients/cobol-lsp-vscode-extension/src/services/snippetcompletion/cobolSnippets.json

```
@@ -1491,7 +1491,7 @@
   "INSPECT TALLYING": {
     "prefix": "INSPECT",
     "body": ["INSPECT ${1:id} "],
```

at least add the `TALLYING` from the snippet title? But perhaps other parameters should be added into the snippet as well.
```suggestion
    "body": ["INSPECT ${1:id} TALLYING"],
```